estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "estate",
+    "version": "1.0",
+    "depends": ["base"],
+    "author": "Kevin Gerard",
```

Comment:
```suggestion
    "author": "Odoo S.A.",
```
We work for Odoo S.A. :smile: